pundix/chain.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Pundi X Chain",
```

Comment:
Chain_name should be all lowercase and not have spaces. "pundix" would be appropriate.